public/stylesheets/style.css

```diff
@@ -72,6 +77,9 @@ main {
 
 .logo {
   width: 50px;
+  text-decoration: none;
+  color: white;
+  font-size: 2em;
```

Comment:
Why not `rem`?

Reply:
im adding this to the Michael app :) ill update to rem where the root size makes more sense.